scripts/check-broken-links.mjs

```diff
@@ -3,17 +3,60 @@ import { LinkChecker } from 'linkinator'
 import chalk from 'chalk'
 import {
   getSitemapLocations,
-  splitPagesIntoSegments,
+  splitPagesIntoBatches,
+  getPagesSlice,
 } from '../playwright/utils/getSitemapLocations.js'
 import fs from 'fs'
 
 const OPTIONS = {
   sitemapUrl: process.env.SITE_URL || 'http://www.va.gov/',
-  batchSize: process.env.BATCH_SIZE || 32,
+  // totalInstances is the number of instances to split work among.
+  totalInstances: process.env.TOTAL_INSTANCES || 1,
+  // instanceNumber is the specifc instance currently running the check.
+  // In this and totalInstances case, the defaults will check the entire set.
+  instanceNumber: process.env.INSTANCE_NUMBER || 1,
+  // batchSize is the number of pararllel link check processes to run.
+  batchSize: process.env.BATCH_SIZE || 20,
   verbose: process.env.VERBOSE || false,
   skipImageLinks: process.env.SKIP_IMAGES || false,
 }
 
+// List of patterns to skip checking. Most of these are to avoid false positives
+// from servers that disallow bot traffic like ours.
+// These can be regular expressions; no quotes around them in that case.
+const LINKS_TO_SKIP = [
```

Comment:
I do not love this mechanism at all, but it's necessary. It would be nice in the future to figure out how to check links on most of these domains successfully, rather than needing to exclude them, since this means we will never find legitimate broken links on these domains.